app/code/Magento/QuoteGraphQl/Model/Cart/SetBillingAddressOnCart.php

```diff
@@ -111,6 +111,45 @@ private function createBillingAddress(
                 (int)$context->getUserId()
             );
         }
+
+        $this->validateAddress($billingAddress);
+
         return $billingAddress;
     }
+
+    /**
+     * Validate quote address.
+     *
+     * @param Address $shippingAddress
+     *
+     * @throws GraphQlInputException
+     */
+    private function validateAddress(Address $shippingAddress)
+    {
+        $errors = $shippingAddress->validate();
+
+        if (true !== $errors) {
+            throw new GraphQlInputException(
+                __('Shipping address error: %message', ['message' => $this->getAddressErrors($errors)])
+            );
+        }
+    }
+
+    /**
+     * Collecting errors.
+     *
+     * @param array $errors
+     * @return string
+     */
+    private function getAddressErrors(array $errors): string
```

Comment:
Please `@todo` with reference to https://github.com/magento/graphql-ce/issues/970

Reply:
Done.